util/design/gen-otp-creator-manuf-state-json.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
```

Comment:
Could this just be combined with the `gen-otp-immutable-rom-ext-json.py`? Specifically, what if you updated the `gen-otp-immutable-rom-ext-json.py` script to take in an additional parameter, i.e., the intended fixed byte of the manuf state field, and then let the `gen-otp-immutable-rom-ext-json.py` script generate both the immutable ROM_EXT and manuf_state fields of the `CREATOR_SW_CFG` partition? that way the `manuf_state` field is only autogenerated by a script, instead of defined manually and autogenerated by this script. Thoughts?

Reply:
I initially thought it might be better to keep the `MANUF_STATE` generation separate. However, it sounds like letting `gen-otp-immutable-rom-ext-json.py` generate `MANUF_STATE` makes more sense. I'm happy to combine the scripts. I'll mark the PR as WIP while I make the changes.

Reply:
Thanks @anthonychen1251 !

Reply:
Done.